### ISSUE TITLE
Expo for web instructions not working

### ISSUE THREAD

**Issue by Nemesisesq** [expo/web-examples#29]
I am trying to follow the expo example [here](https://github.com/expo/web-examples/blob/master/expo-managed-workflow/README.md)

I added react native web and react dom and expo@next but I get the following error below.

What am I doing wrong?

[23:41:43] Starting project at /Users/nem/parcl/parcl
[23:41:43] Expo DevTools is running at http://localhost:19002
[23:41:45] Error: Problem validating fields in app.json. See https://docs.expo.io/versions/v32.0.0/workflow/configuration/
[23:41:45]  • Field: platforms - should NOT have more than 2 items.
[23:41:47] Starting Metro Bundler on port 19004.
[23:41:48] Cannot find module './validators/Presets'
[23:41:48] Set EXPO_DEBUG=true in your env to view the stack trace.

**Comment by z0al** [comment]
Same, here is my log:

```
> expo start --web --non-interactive

[07:58:16] Starting project at /home/z0al/Desktop/expoapp
[07:58:17] Expo DevTools is running at http://localhost:19002
[07:58:18] Error: Problems validating fields in app.json. See https://docs.expo.io/versions/v32.0.0/workflow/configuration/
[07:58:18]  • Field: platforms - should NOT have more than 2 items.
[07:58:18]  • Field: platforms[0] - 'platforms[0]' should match pattern "^(ios|android)$".
[07:58:19] Starting Metro Bundler on port 19001.
[07:58:20] Cannot find module './validators/Presets'
[07:58:20] Set EXPO_DEBUG=true in your env to view the stack trace.
npm ERR! code ELIFECYCLE
```

It seems SDK v32.0.0 does some validation not to use unsupported platform.

**Comment by foloinfo** [comment]
Do you use `"expo": "^33.0.0-alpha.web.1"` ?
It shows me these `should NOT have more than 2 items` error, but it works fine for me.

my (partial) package.json

```
  "dependencies": {
    "expo": "^33.0.0-alpha.web.1",
    "react": "16.5.0",
    "react-dom": "^16.8.6",
    "react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
    "react-native-web": "^0.11.2",
    "react-navigation": "3.5.0-alpha.0",
  },
```

**Comment by z0al** [comment]
@foloinfo I did, but for some reason, it didn't work. Maybe my `node_modules` folder was messed up.

I removed everything and started a fresh blank project and it works now. Here is my `package.json`.

```json
{
	"main": "node_modules/expo/AppEntry.js",
	"scripts": {
		"start": "expo start --web --non-interactive",
		"android": "expo start --android",
		"ios": "expo start --ios",
		"eject": "expo eject"
	},
	"dependencies": {
		"expo": "^33.0.0-alpha.web.1",
		"react": "16.5.0",
		"react-dom": "^16.8.6",
		"react-native": "https://github.com/expo/react-native/archive/sdk-32.0.0.tar.gz",
		"react-native-web": "^0.11.2"
	},
	"devDependencies": {
		"babel-preset-expo": "^5.1.1"
	},
	"private": true
}
```

**Comment by z0al** [comment]
Ok, I got it now it worked for me on macOS (at work), but not in Linux (my fav OS at home). 

If you're wondering why? cause unlike GNU/Linux, macOS is case-insensitive, so [this](https://github.com/expo/expo-cli/blob/bb9f9a63d5bb6b6b8cef2d35044b70cda8f82f73/packages/webpack-pwa-manifest-plugin/src/index.js#L11) import would work there even though the actual file name is "presets" not "Presets".

```javascript
import validatePresets from './validators/Presets';
```

The fix is obvious and it actually worked when I modified the `node_modules` manually :)
